backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -730,12 +692,11 @@ export class ContributionResolver {
     contributionToUpdate.deniedAt = new Date()
     const res = await contributionToUpdate.save()
 
-    const event = new Event()
-    const eventAdminContributionDeny = new EventAdminContributionDeny()
-    eventAdminContributionDeny.userId = contributionToUpdate.userId
-    eventAdminContributionDeny.amount = contributionToUpdate.amount
-    eventAdminContributionDeny.contributionId = contributionToUpdate.id
-    await writeEvent(event.setEventAdminContributionDeny(eventAdminContributionDeny))
+    await EVENT_ADMIN_CONTRIBUTION_DENY(
+      moderator.id,
```

Comment:
I do not agree with this change. Yes, we do not have events of this kind so far. But the user stored in the database should be the one the action affects.

Reply:
On admin delete we have stored the affected user as well. For consistency reasons, I want to see the user here as well